tools/common/__init__.py

```diff
@@ -91,20 +92,18 @@ def print_device(device: BlockDevice | RpcDevice) -> None:
 
 def print_block_device(device: BlockDevice) -> None:
     """Print block (GEN1) device data."""
-    assert device.blocks
+    if TYPE_CHECKING:
+        assert device.blocks
 
     for block in device.blocks:
         print(block)
         for attr, value in block.current_values().items():
             info = block.info(attr)
 
             if value is None:
-                value = "-"
+                value = "-"  # noqa: PLW2901
```

Comment:
Would be nice to replace it with a temp value (`_value`) and use it for the print
`_value = value if value is not None else "-"`